editor/animation_bezier_editor.cpp

```diff
@@ -1670,6 +1681,17 @@ void AnimationBezierTrackEdit::_menu_selected(int p_index) {
 				EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
 				undo_redo->create_action(TTR("Add Bezier Point"));
 				undo_redo->add_do_method(animation.ptr(), "track_insert_key", selected_track, time, new_point);
+				int k_idx = animation->track_find_key(selected_track, time) + 1;
+				switch (int(EDITOR_GET("editors/animation/default_bezier_key_behavior"))) {
```

Comment:
Same above.

Reply:
done. maybe you can check if I did it the right way.